app/src/main/java/org/p2p/wallet/striga/model/StrigaDataLayerResult.kt

```diff
@@ -1,8 +1,8 @@
 package org.p2p.wallet.striga.model
 
-sealed interface StrigaDataLayerResult<out T> {
-    data class Success<T>(val value: T) : StrigaDataLayerResult<T>
-    class Failure<T>(val error: StrigaDataLayerError) : StrigaDataLayerResult<T>
+sealed class StrigaDataLayerResult<out T> {
```

Comment:
changed to `class` because cannot use inline functions within non-final interfaces